readthedocs/rtd_tests/tests/test_privacy_urls.py

```diff
@@ -364,7 +364,7 @@ def setUp(self):
             'api_webhook_gitlab': {'status_code': 405},
             'api_webhook_bitbucket': {'status_code': 405},
             'api_webhook_generic': {'status_code': 403},
-            'domaindata-detail': {'status_code': 404},
+            'sphinx_domains-detail': {'status_code': 404},
```

Comment:
I think it doesn't matter too much, but to keep consistency here with the other endpoints, I'd say that it should be called `sphinxdomains-detail` (without the `_`).

Reply:
This is autogenerated by REST framework.

Reply:
Well, yes... but it comes from the `register`: https://github.com/rtfd/readthedocs.org/pull/5289/files/021e12d9472f3e4cb1f7f69413e0af0e25808c86..ab7a30cf1cb40012ce71704824723aa5c1597b67#diff-48d1599aaf2a6be451a9f14bdf0cd81aR38

I should have commented that other line instead :)

Reply:
Agreed. It also shouldn't be plural. Fixed.